obfx_modules/gutenberg-blocks/init.php

```diff
@@ -34,10 +34,7 @@ public function __construct() {
 	 * @return bool
 	 */
 	public function enable_module() {
-		require_once( ABSPATH . 'wp-admin' . '/includes/plugin.php' );
-		// TODO: Fix this check before gutenberg is merged into WordPress core.
-		// checking function_exists( 'register_block_type' ) returns false at first and true later in the code.
-		return is_plugin_active( 'gutenberg/gutenberg.php' );
+		return true;
```

Comment:
@abaicus i think we should check if the wordpress version is at least 5.0

Reply:
@selul Or if the previous condition was true? So that it would work even on non-udated versions?

Reply:
yep, makes sense